app/global-translations/locale-en.json

```diff
@@ -786,6 +786,7 @@
     "label.heading.accountusage": "Account Usage",
     "label.heading.viewaccounttransferdetails": "View Account Transfer Details",
     "label.heading.transactiondetails": "Transaction Details",
+    "label.heading.paymentdetails": "Payment Details:",
```

Comment:
yes sir* that label key is already present sir, sorry sir i dint check. can
i modify and send pull request again sir.

On Mon, Apr 13, 2015 at 1:40 PM, Rajesh Reddy <
rajesh.reddy@confluxtechnologies.com> wrote:

> yes sit that label key is already present sir, sorry sir i dint check. can
> i modify and send pull request again sir.
>
> On Mon, Apr 13, 2015 at 1:32 PM, Vishwas Babu A J <
> notifications@github.com> wrote:
>
>> In app/global-translations/locale-en.json
>> <https://github.com/openMF/community-app/pull/1327#discussion_r28220258>:
>>
>> > @@ -786,6 +786,7 @@
>> >      "label.heading.accountusage": "Account Usage",
>> >      "label.heading.viewaccounttransferdetails": "View Account Transfer Details",
>> >      "label.heading.transactiondetails": "Transaction Details",
>> > +    "label.heading.paymentdetails": "Payment Details:",
>>
>> this label key already exists
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/openMF/community-app/pull/1327/files#r28220258>.
>>
>
>
>
> --
> Thank you & regards
> Rajesh Reddy k
> 7795405419
>



-- 
Thank you & regards
Rajesh Reddy k
7795405419